src/main/java/org/traccar/directions/timeDistance/TimeDistanceRequest.java

```diff
@@ -0,0 +1,46 @@
+package org.traccar.directions.timeDistance;
```

Comment:
Don't forget to add copyright header to each file (except unit tests). You can put your name if you want.